src/devices/tuya.ts

```diff
@@ -7427,6 +7427,106 @@ export const definitions: DefinitionWithExtend[] = [
             await reporting.bind(device.getEndpoint(4), coordinatorEndpoint, ["genOnOff"]);
         },
     },
+    {
+        fingerprint: tuya.fingerprint("TS0726", ["_TZ3000_qhyadm57"]),
```

Comment:
Please merge these with the already existing `TS0726_2_gang` (same goes for the others)